src/node_crypto.cc

```diff
@@ -1323,6 +1330,120 @@ void SecureContext::GetCertificate(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+#ifdef OPENSSL_PSK_SUPPORT
+
+void SecureContext::SetPskIdentity(const FunctionCallbackInfo<Value>& args) {
+  SecureContext* wrap = Unwrap<SecureContext>(args.Holder());
+
+  if (args.Length() != 1 || !args[0]->IsString()) {
```

Comment:
Not convinced that this should error if more than 1 argument is provided. Perhaps just `if (!args[0]->IsString()) {`